docs/how_to_guides/validation/how_to_add_a_validation_operator.rst

```diff
@@ -0,0 +1,74 @@
+.. _how_to_guides__validation__how_to_add_a_validation_operator:
+
+How to add a Validation Operator
+======================================
+
+This guide will help you add a new instance of a :ref:`Validation Operator <validation_operators_and_actions>`. Validation Operators give you the ability to validate multiple batches
+of data together. Additionally, each Validation Operator encodes a set of business rules, such as performing various actions based on the results of validation. When you add an instance of a Validation Operator,
+you can use its business rules in your project.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+  - :ref:`Set up a working deployment of Great Expectations <getting_started>`.
+  - Created at least one Expectation Suite.
+  - Created at least one :ref:`Checkpoint <how_to_guides__validation__how_to_create_a_new_checkpoint>`. You will need it in order to test that your new Validation Operator is working.
+
+Steps
+-----
+
+The snippet below shows a portion of your ``great_expectations.yml`` configuration after you perform the following steps. The steps will explain each line in this snippet.
+
+.. code-block:: yaml
+   :linenos:
+
+   validation_operators:
+     action_list_operator:
+       class_name: ActionListValidationOperator
+       action_list:
+       - name: store_validation_result
+         action:
+           class_name: StoreValidationResultAction
+       - name: store_evaluation_params
+         action:
+           class_name: StoreEvaluationParametersAction
+       - name: update_data_docs
+         action:
+           class_name: UpdateDataDocsAction
+     # Next Validation Operator was added manually
+     my_second_validation_operator:
+       class_name: ActionListValidationOperator
+       action_list:
+       - name: store_validation_result
+         action:
+           class_name: StoreValidationResultAction
+
+1. Open your project's ``great_expectations.yml`` configuration file and navigate to the ``validation_operators`` section (line 1 in the snippet). This section contains the Validation Operator instance ``action_list_operator`` that was automatically created by the ``great_expectations init`` CLI command.
+2. Add a new block after the existing Validation Operator instances. The name of the block is the name you are giving to the new Validation Operator instance (line 15 in the snippet). These names must be unique within a project.
+3. Add a ``class_name`` attribute in the new block you added in the previous step (line 16 in the snippet). The value is the name of the class that implements the Validation Operator that you are adding. This can be one of the classes that are included in Great Expectations or a class that you implemented. This example adds another instance of :py:class:`great_expectations.validation_operators.validation_operators.ActionListValidationOperator`.
```

Comment:
This doesn't follow our style of short bolded sentence followed by description.

Reply:
I was going to let this slide, but you're right.

How about these headers? (Eugene's text can become indented paragraphs without other changes.)
1. Find the validation_operators section in your great_expectations config file.
2. Add a new Validation Operator block.
3. Pick a class_name
4. Configure in other fields
5. Test your configuration

Reply:
Addressed